tests/integration/tx/decode_test.go

```diff
@@ -160,5 +161,5 @@ func (d dummyAddressCodec) StringToBytes(text string) ([]byte, error) {
 }
 
 func (d dummyAddressCodec) BytesToString(bz []byte) (string, error) {
-	return string(bz), nil
+	return common.Bytes2Hex(bz), nil
```

Comment:
Similar question as one of the comments above: Why we didn't do `common.FromHex()` in `StringToBytes` ? Probably it didn't affect any of the tests and we wanted to keep the changes minimal.

Reply:
Pushed 🙌